luigi/worker.py

```diff
@@ -389,11 +389,14 @@ def __init__(self, scheduler=None, worker_id=None, worker_processes=1, assistant
         self.host = socket.gethostname()
         self._scheduled_tasks = {}
         self._suspended_tasks = {}
+        self._batch_running_tasks = {}
+        self._batch_classes_sent = set()
```

Comment:
Maybe rename to `_batch_families_sent`? And/or add comment `  # set of task families`